README.md

```diff
@@ -136,6 +138,83 @@ Writing a new benchmark fixture would generally look like:
 
 The `benchmark_time` fixture provides a fairly simple example.
 
+## Investigating performance regressions
+
+It is not always obvious what the cause of a seeming performance regression is.
+It could be due to a new version of a direct or transitive dependency,
+and it could be due to a change in the Coiled platform.
+But often it is due to a change in `dask` or `distributed`.
+If you suspect that is the case, Coiled's `package_sync` feature combines well with the benchmarking infrastructure here and `git bisect`.
+
+The following is an example workflow which could be used to identify a specific commit in `dask` which introduced a performance regression.
+
+#### Create your software environment
+
+You should create a software environment which can run this test suite, but with an editable install of `dask`:
+```bash
+conda env create -n test-env --file ci/environment.yml
+conda activate test-env
+pip install .
```

Comment:
this part is confusing. Are we suposed to do `pip install .` in the runtime repository, is the step below is the one we actually want to do?